v8/mastodoncomments/templates/jinja/comments_helper_mastodoncomments.tmpl

```diff
@@ -0,0 +1,31 @@
+{#  -*- coding: utf-8 -*- #}
+{% macro comment_form(url, title, identifier) %}
+{% if comment_system_id and post.meta('mastodon') %}
+
+<p>With an account on the Fediverse or Mastodon, you can respond to this <a href="https://{{ comment_system_id }}/@{{ mastodon_config['account'] }}/{{ post.meta('mastodon') }}">post</a>. Since Mastodon is decentralized, you can use your existing account hosted by another Mastodon server or compatible platform if you don't have an account on this one. Known non-private replies are displayed below.</p>
+
+<p id="mastodon-comments-list"><button id="load-comment">Load comments</button></p>
+<div id="comments-wrapper">
+<noscript><p>Loading comments relies on JavaScript. Try enabling JavaScript and reloading, or visit <a href="https://{{ comment_system_id }}/@{{ mastodon_config['account'] }}/{{ post.meta('mastodon') }}">the original post</a> on Mastodon.</p></noscript>
+</div>
+<noscript>You need JavaScript to view the comments.</noscript>
+
+<script type="text/javascript" src="/js/purify.min.js"></script>
```

Comment:
Please adjust the templates and README to use the `/assets/js` and `/assets/css` folders.

Reply:
I want to use `<script type="text/javascript" src="/assets/js/purify.min.js"></script>` and `files/assets/js/purify.min.js` — so we keep all CSS and JS in the assets folder (as Nikola does).